packages/core/src/common/utils/keyboardUtils.ts

```diff
@@ -25,3 +25,21 @@ export function isKeyboardClick(event: React.KeyboardEvent<HTMLElement>) {
 export function isArrowKey(event: React.KeyboardEvent<HTMLElement>) {
     return ["ArrowUp", "ArrowDown", "ArrowLeft", "ArrowRight"].indexOf(event.key) >= 0;
 }
+
+/**
+ * Direction multiplier for component such as radiogroup, tablist
+ *
+ * @param upMovesLeft
+ * If true, up arrow returns same as left arrow, down arrow returns same as right arrow.
+ * If false, down arrow returns same as left arrow, up arrow returns same as right arrow.
+ * @returns -1 for left, 1 for right, undefined if not an arrow keypress.
+ */
+export function getArrowKeyDirection(event: React.KeyboardEvent<HTMLElement>, upMovesLeft: boolean = true) {
```

Comment:
`upMovesLeft` is interesting - are we currently following expected behavior correctly? it definitely makes sense in the slider case for up to move right but I'm wondering if up should always be to the right

this feels like something that may change based on locale, possibly in addition to component usage

not sure we need to figure this out now since this keeps existing behavior but leaving this comment for posterity at least

Reply:
- `Slider`: Definitely makes sense for `up = right`.

- `Tabs`: the aria `tabs` example doesn't have any effect from the up/down arrow keys, only left/right: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/ . So if anything we should remove the effect of an up/down keypress. But with this code, the existing implementation goes unchanged.

- `SegmentedControl`: since we are giving this the `radiogroup` role, here in the official example of this role you can see that `up = left`: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/